externs/shaka/player.js

```diff
@@ -536,6 +537,10 @@ shaka.extern.DrmConfiguration;
  * @property {number} defaultPresentationDelay
  *   A default presentationDelay if suggestedPresentationDelay is missing
  *   in the MPEG DASH manifest. This has to be bigger than minBufferTime * 1.5.
+ * @property {boolean} ignoreMinBufferTime
+ *   If true will cause DASH parser to ignore minBufferTime from manifest
```

Comment:
Minor English nit: This sentence needs a period at the end, and the next line should begin with a capital letter.